TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst

```diff
@@ -17,6 +17,7 @@ TYPO3.TypoScript:Array
 Render multiple nested definitions and concatenate the results.
 
 :[key]: (string) A nested definition (simple value, expression or object) that evaluates to a string
+:[key].@ignoreProperties: (array) A list of properties to ignore from being "rendered" during evaluation
```

Comment:
Small nitpick: Why quotes here?

Wouldn't "properties to ignore from being evaluated during rendering" be correct?